### ISSUE TITLE
Add license/attribution for the music used in this template if applicable

### ISSUE THREAD

**Issue by lungsin** [bitbrain/godot-gamejam#11]
I wasn't able to find the source of the music used in this file, specifically [corgie-title.ogg](https://github.com/bitbrain/godot-gamejam/blob/main/godot/music/corgie-title.ogg). Just want to make sure if attribution is required to use it in a gamejam.

**Comment by bitbrain** [comment]
Good catch - I was not intending to ship the music as part of this. It's the title soundtrack for [CORG-E](https://bitbrain.itch.io/corg-e], a game I built with a friend. The music is by him.

I'll remove the soundtrack.
